src/execute/swagger2/build-request.js

```diff
@@ -84,7 +84,8 @@ export function applySecurities({request, securities = {}, operation = {}, spec}
           }
         }
         else if (type === 'oauth2' && accessToken) {
-          result.headers.authorization = `${tokenType || 'Bearer'} ${accessToken}`
+          tokenType = (!tokenType || tokenType === 'bearer') ? 'Bearer' : tokenType
```

Comment:
I'd prefer for the `token_type` detection for bearer to be case-insensitive, i.e. our `Bearer` override will kick in for token_type `bearer`, `BEARER`, `BeArEr`.

I know this is a minor point, and there may not be an OAuth server implementation that sends those values, but if we're not paying attention to token_type `bearer` vs `Bearer`, I'd like to go all the way with it, instead of adding a new magic token_type that triggers the correct header value.

Reply:
Updated